tests/phpunit/tests/theme/wpThemeJsonResolver.php

```diff
@@ -455,15 +458,18 @@ public function test_add_theme_supports_are_loaded_for_themes_without_theme_json
 		);
 		add_theme_support( 'editor-color-palette', $color_palette );
 		add_theme_support( 'custom-line-height' );
+		add_theme_support( 'appearance-tools' );
 
 		$settings = WP_Theme_JSON_Resolver::get_theme_data()->get_settings();
 
 		remove_theme_support( 'custom-line-height' );
 		remove_theme_support( 'editor-color-palette' );
+		add_theme_support( 'appearance-tools' );
```

Comment:
Should this be `remove_theme_support`?

```suggestion
		remove_theme_support( 'appearance-tools' );
```

Reply:
oh whoops yes it should 😅